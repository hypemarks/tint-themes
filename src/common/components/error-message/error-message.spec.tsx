import { render } from '@testing-library/react';
import React from 'react';
import ErrorMessage from './error-message';

describe('Test <ErrorMessage /> Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(<ErrorMessage />);
    const errorMessage = wrapper.queryByTestId('error-message');
    expect(errorMessage).toHaveTextContent('Something went wrong. Please contact support.');
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
