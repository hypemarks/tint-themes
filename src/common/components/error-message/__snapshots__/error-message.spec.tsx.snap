// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <ErrorMessage /> Component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="error-message"
    data-testid="error-message"
  >
    <h1>
      Something went wrong. Please contact support.
    </h1>
  </div>
</DocumentFragment>
`;
